Kernel/Arch/x86_64/init.cpp

```diff
@@ -10,12 +10,13 @@
 #include <Kernel/Arch/x86_64/Hypervisor/VMWareBackdoor.h>
 #include <Kernel/Arch/x86_64/Interrupts/APIC.h>
 #include <Kernel/Arch/x86_64/Interrupts/PIC.h>
-#include <Kernel/BootInfo.h>
+#include <Kernel/Boot/BootInfo.h>
+#include <Kernel/Boot/CommandLine.h>
+#include <Kernel/Boot/Multiboot.h>
 #include <Kernel/Bus/PCI/Access.h>
```

Comment:
not sure: should `Bus` go under `Devices` too?

Reply:
Not sure this is a good idea. Devices is just a generic concept, not tied to buses, but also the opposite can't be said (buses are not tied to devices). I rather keep it this way.